tools/mo/openvino/tools/mo/front/tf/loader.py

```diff
@@ -226,6 +226,14 @@ def tf_function(x):
 
         conc_func = tf_function.get_concrete_function(model_inputs)
         return freeze_tf2_concrete_function(model, conc_func, env_setup)
+    if isinstance(model, tf.types.experimental.GenericFunction):
+        env_setup = get_environment_setup("tf")
+
+        assert hasattr(model, "input_signature") and model.input_signature is not None, \
+            "'input_signature' needs to be set for model conversion."
```

Comment:
can we have a test for this case with `input_signature` when obtaining of `ConcreteFunction` from `GenericFunction` is not possible

Reply:
Done.